src/Perpetuum/Robots/Robot.cs

```diff
@@ -353,14 +353,6 @@ public void CreateComponents()
             }
         }
 
-        protected override bool IsDetected(Unit target)
-        {
-            if (_lockHandler.IsLocked(target))
```

Comment:
This was the logic that we seem to be at odds with.
Looks completely intentional... scary